openj9-11-rhel8.yaml

```diff
@@ -4,7 +4,7 @@
 
 schema_version: 1
 
-from: "ubi8:8-released"
+from: "registry.access.redhat.com/ubi8/ubi-minimal"
```

Comment:
That's not a 1-to-1 change. ubi8 => ubi-minimal. Are you sure? It would mandate micro-dnf.

Reply:
I'd suggest: `registry.access.redhat.com/ubi8`

Reply:
That's a good catch!

Reply:
resolving with d20c9fc